### ISSUE TITLE
My Site: VoiceOver: site icon is not accessible

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12179]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. Go to My Sties and select a site.
1. Try to double tap the site icon at the top next to the site title and tagline.

**Result:** it's not possible to access the site icon with VoiceOver enabled. ([37s](https://cloudup.com/cW_7txdP1qw))

<img src="https://user-images.githubusercontent.com/1119271/61652790-02868700-ac76-11e9-8c00-b5b2a3d7c12c.png" width="270" alt="site-icon-is-not-accessible">
<sup>Tested with WPiOS 12.8.1 on iPhone 6S iOS 12.3.1.</sup>

<sub>(internal reference: p77Llu-bQ6-p2 h/t @etoledom)</sub>
